applications/FluidDynamicsApplication/python_scripts/apply_two_fluids_inlet_process.py

```diff
@@ -0,0 +1,161 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
```

Comment:
```suggestion

```

seems not to be used

Reply:
It is used in `self.two_fluid_inlet_process = KratosFluid.TwoFluidsInletProcess( self.inlet_model_part, settings["interface_settings"], self.variational_distance_process )`, once.